node-template/build.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
```

Comment:
This license header conflicts with the LICENSE in this directory (two files above), doesn't it?